lib/ReportDataCollector.php

```diff
@@ -474,7 +474,7 @@ private function getPhpInfoDetailArray() {
 
 		// Get the phpinfo, parse it, and record it (parts from http://www.php.net/manual/en/function.phpinfo.php#87463)
 		\ob_start();
-		\phpinfo(-1);
+		\phpinfo(INFO_ALL & ~INFO_ENVIRONMENT);
```

Comment:
LGTM when taking https://github.com/owncloud/enterprise/issues/5507#issuecomment-1396767934 in to account as @enbrnz has more practical experience here.  

Could we potentially still loose some bits from the Apache env (?). But from my experience the cases where this info leads to a solution are very rare.

There is a [list](https://github.com/owncloud-docker/base/blob/master/ENVIRONMENT.md) of all supported env-vars so we could in theory filter based on it. But I might be overthinking this. :microscope:

Reply:
For ownCloud specific variables, they should appear in the ownCloud's configuration (either config.php or in the appconfig table). Any additional information from the environment should be picked from the actual configuration of the service; this should also include apache.